mindmeld-ui/src/recognition/languages.js

```diff
@@ -0,0 +1,482 @@
+export default [
+  /*
```

Comment:
Should we uncomment these languages?

Reply:
Or maybe we can just remove them.

Reply:
Removed the whole file, turns out we were not using it!